idowuadeleke/BANKA
# #1655539588 refactor code base

### What does this PR do?
refactor codebase

### Description of Task to be completed?
- Refactor code to conform to user specifications and eslint requirements

### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ch-165539588-refactor-codebase`

- new code base should meet user specifications and eslint requirements

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#165539588](https://www.pivotaltracker.com/story/show/165539588)

### Screenshots (if appropriate)
N/A
